api/BUILD

```diff
@@ -220,6 +220,8 @@ proto_library(
         "//envoy/extensions/filters/network/thrift_proxy/filters/ratelimit/v3:pkg",
         "//envoy/extensions/filters/network/thrift_proxy/v3:pkg",
         "//envoy/extensions/filters/network/zookeeper_proxy/v3:pkg",
+        "//envoy/extensions/internal_redirect/previous_routes/v3:pkg",
+        "//envoy/extensions/internal_redirect/whitelisted_routes/v3:pkg",
```

Comment:
s/whitelist/allowlist here and elsewhere to align with current naming best practices

Reply:
Done.